explainer.md

```diff
@@ -445,17 +444,19 @@ Similarly, the `deactivate` event can be used to detect when the user removes th
 
 ```js
 vrDevice.addEventListener('deactivate', vrDeviceEvent => {
-  if (vrDevice.activeSession) {
-    vrDevice.activeSession.endSession().then(OnSessionEnded);
+  if (vrSession) {
+    vrSession.endSession().then(OnSessionEnded);
   }
 });
 ```
 
 ### Responding to a suspended session
```

Comment:
It may help address comments above and below to add a separate section that discusses when a UA should suspend a session. That could be referenced both above and from here.